go_api/document.go

```diff
@@ -0,0 +1,17 @@
+package main
+
+type UserRegister struct {
+	Name  string `form:"name"`
+	Phone string `form:"phone"`
+}
+
+type UserLogin struct {
+	Phone    string `form:"phone"`
+	Password string `form:"password"`
+}
+
+type User struct {
+	Phone    string `form:"phone" bson:"phone" json:"phone"`
+	Name     string `form:"name" bson:"name" json:"name"`
+	Password string `form:"password" bson:"password" json:"password"`
+}
```

Comment:
If we were to build the ability to send an email manually we could have a struct with `to`, `subject`, `body`, and maybe `signature`

Then we would be able to make a nice Svelte form with those fields

Reply:
Yeah gonna put it inside a struct :D

Reply:
Done